src/conf.R

```diff
@@ -569,7 +569,10 @@ Fields$tangent_name = add.to.var.name(Fields$name,"d")
 
 Fields$area = (Fields$maxx-Fields$minx+1)*(Fields$maxy-Fields$miny+1)*(Fields$maxz-Fields$minz+1)
 Fields$simple_access = (Fields$area == 1)
-Fields$big = Fields$area > 27
+# NOTE: This heuristic does not always work, currently enabled only for fields
+# which have more dynamic access pattern
+# Fields$big = Fields$area > 27
+Fields$big = ifelse(Fields$name %in% c("gradPhiVal_x", "gradPhiVal_y", "gradPhiVal_z", "gradPhi_PhaseF"), TRUE, FALSE)
```

Comment:
This has to be non-model specific. Best if it could be selected some way at "AddField/AddDensity".

Reply:
Yes, I know. I was planning to add additional flag to AddDensity / AddField

Reply:
@llaniewski, do we want to keep the default setting `Fields$big = Fields$area > 27` ?  

I also am going to change the name of this settting from `big` to `optimise_for_static_access`, unless you have a better name suggestion